Source/SwiftLintFramework/Extensions/Configuration+LintableFiles.swift

```diff
@@ -14,6 +14,18 @@ extension Configuration {
         return lintablePaths(inPath: path).flatMap(File.init(path:))
     }
 
+    public func lintableFiles(ofFiles files: [File]) -> [File] {
+        let excludedPaths = excluded.flatMap {
+            FileManager.default.filesToLint(inPath: $0, rootDirectory: rootPath)
```

Comment:
This doesn't inject the `LintableFileManager` like we do everywhere else we interact with `FileManager`.